itou/templates/layout/base.html

```diff
@@ -68,16 +68,56 @@
                             La version de votre navigateur n'est plus supportée. Veuillez utiliser une version plus récente pour améliorer votre expérience sur notre site.
                         </p>
                     </div>
-
                 {% endblock %}
             </div>
 
             <div class="container container-breadcrumb">
                 {% block breadcrumbs %}{% endblock %}
             </div>
 
-            {% block content %}{% endblock %}
+            {% comment %}
+
+                METHODE 01
+                Pros : Si les blocks {% content_title_wrapper %} et {% content_title %} ne sont pas présents dans les templates enfants, la partie de code du section/container html ci-dessous (inclus dans le {% content_title_wrapper %}) n'est pas affichée vide (html plus propre et plus a11y friendly)
+                Cons : Code dans les templates enfants un peu lourd, nécessitant 2 blocks et une variable en with (voir exemple ici itou/templates/static/legal/notice.html)
+
+                {% block content_title_wrapper %}
+                    {% if content_title %}
+                        <section class="s-title-01">
+                            <div class="s-title-01__container container">
+                                <div class="s-title-01__row row">
+                                    <div class="s-title-01__col col-12">
+                                        {% block content_title %}{% endblock %}
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    {% endif %}
+                {% endblock %}
+
 
+                METHODE 02
+                Pros : Code dans les templates enfants plus léger (voir exemple ici itou/templates/static/legal/notice.html)
+                Cons : Obligation de mettre le {% block content_title_wrapper %} vide dans les templates enfants pour surcharger le code du section/container ci-dessous
+
+                {% block content_title_wrapper %}
+                    <section class="s-title-01">
+                        <div class="s-title-01__container container">
+                            <div class="s-title-01__row row">
+                                <div class="s-title-01__col col-12">
+                                    {% block content_title %}{% endblock %}
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                {% endblock %}
```

Comment:
Hum, je n'arrive pas vraiment à choisir. Idéalement, il faudrait vérifier si un bloc a été défini et, s'il l'est, alors afficher le wrapper. Quelque chose dans ce style-là :
```
{% if content_title %}
                    <section class="s-title-01">
                        <div class="s-title-01__container container">
                            <div class="s-title-01__row row">
                                <div class="s-title-01__col col-12">
                                    {% block content_title %}{% endblock %}
                                </div>
                            </div>
                        </div>
                    </section>
{% endif %}
```

Je suis tombée [sur cet article qui fait exactement cela](https://chase-seibert.github.io/blog/2010/10/01/check-if-a-block-is-defined-in-django.html) mais il a 13 ans ! :scream: Je me sens vieille ! :laughing: 

Là je dois partir mais je continuerai de regarder demain, c'est très intéressant.